### PR TITLE
Moves MultiLanguageString from sirius-db to this project and performs some simplifications.

### PR DESCRIPTION
**Pull request by andyHa**

### CODE REVIEW

File: src/main/java/sirius/biz/translations/MultiLanguageString.java [hunk 1]
```diff
@@ -24,16 +25,19 @@
 /**
  * Provides a language-text map as property value.
  *
- * @see sirius.db.mixing.properties.MultiLanguageStringProperty
+ * @see MultiLanguageStringProperty
  */
 public class MultiLanguageString extends SafeMap<String, String> {
 
+    /**
+     * Represents tha fake language code used to store the fallback value.
```
Comment (sabieber) [inline src/main/java/sirius/biz/translations/MultiLanguageString.java:33]: tha -> the

File: src/main/java/sirius/biz/web/ComplexLoadProperty.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ */
+
+package sirius.biz.web;
+
+import sirius.db.mixing.BaseEntity;
+import sirius.web.http.WebContext;
+
+/**
+ * Marks {@link sirius.db.mixing.Property properties} which are aware of {@link WebContext web contexts} and handle
+ * their save logik themselves.
```
Comment (sabieber) [inline src/main/java/sirius/biz/web/ComplexLoadProperty.java:16]: logik -> logic

**Review by sabieber — CHANGES_REQUESTED** [review]

File: src/test/resources/test_de.properties [hunk 3]
```diff
@@ -3,3 +3,4 @@ MongoPageHelperEntity.stringField = String Feld
 ElasticPageHelperEntity.booleanField = Bool Feld
 ElasticPageHelperEntity.stringField = String Feld
 PrefixSearchableTestEntity.multiLanguageText = Multi Language Text
+MongoMultiLanguageStringRequiredEntity.multiLangText=MultiLanguageTextRequired
```
Comment (idlira) [inline src/test/resources/test_de.properties:6]:
```suggestion
MongoMultiLanguageStringRequiredEntity.multiLangText = MultiLanguageTextRequired
```

File: src/main/java/sirius/biz/translations/MultiLanguageString.java [hunk 4]
```diff
@@ -0,0 +1,397 @@
...
+
+/**
+ * Provides a language-text map as property value.
+ *
+ * @see MultiLanguageStringProperty
+ */
+public class MultiLanguageString extends SafeMap<String, String> {
+
+    /**
+     * Represents tha fake language code used to store the fallback value.
```
Comment (andyHa) [inline src/main/java/sirius/biz/translations/MultiLanguageString.java:33]:
```suggestion
     * Represents tha fake language code used to store the fallback value.
```

**Review by sabieber — APPROVED** [review]

**Pull request merged**